service_mesh/v2x/ossm-traffic-manage.adoc

```diff
@@ -8,14 +8,14 @@ toc::[]
 
 {SMProductName} lets you control the flow of traffic and API calls between services. Some services in your service mesh may need to communicate within the mesh and others may need to be hidden. You can manage the traffic to hide specific backend services, expose services, create testing or versioning deployments, or add a security layer on a set of services.
```

Comment:
```suggestion
Using {SMProductName}, you can control the flow of traffic and API calls between services. Some services in your service mesh may need to communicate within the mesh and others may need to be hidden. You can manage the traffic to hide specific backend services, expose services, create testing or versioning deployments, or add a security layer on a set of services.
```
- see comment above about attributing agency to inanimate objects

Reply:
Change made